packages/studio-ui/src/utils/getFunctionComponent.ts

```diff
@@ -1,15 +1,7 @@
-import { FunctionComponent } from "react";
 import { ImportType } from "../store/models/ImportType";
 
 export default function getFunctionComponent(
-  importedValue: Record<string, unknown>,
-  name: string
+  importedValue: Record<string, unknown>
 ): ImportType | undefined {
-  if (typeof importedValue[name] === "function") {
```

Comment:
We don't support components that are not default exported, I think we probably used to which is why we had this logic but haven't checked. Possibly is related to the external components support that Tristan did that we had to cut